src/utils/resolveXYScales.js

```diff
@@ -341,16 +337,18 @@ export default function resolveXYScales(ComposedComponent) {
       return margin;
     }
 
-    _makeScales = ({width, height, scaleType={}, domain={}, margin={}, scale={}}) => {
+    _makeScales = ({width, height, scaleType={}, domain={}, margin={}, scale={}, spacing={}}) => {
       const {invertScale, nice, tickCount, ticks} = this.props;
+      
+      const innerMarginWidthX = _.subtract(...innerRangeX(width, margin).reverse());
+      const innerMarginWidthY = _.subtract(...innerRangeY(height, margin));
```

Comment:
I think this is basically a re-implementation of something that already exists - `innerWidth` and `innerHeight` from `Margin.js`. You should just be able to do `const innerMarginWidthX = innerWidth(width, margin)` and same with height.

Also the names are kinda funky... sounds like they are measuring the size of the margin, rather than the stuff inside. Also "Y width" is usually called "height" 😛. Maybe `innerChartWidth` and `innerChartHeight`?